src/Pact/Analyze/Types/Shared.hs

```diff
@@ -14,6 +14,7 @@
 {-# LANGUAGE Rank2Types                 #-}
 {-# LANGUAGE ScopedTypeVariables        #-}
 {-# LANGUAGE StandaloneDeriving         #-}
+{-# LANGUAGE StrictData                 #-}
```

Comment:
did you mean for this to be `Strict` instead?

Reply:
fixed